lib/standard/exec.nit

```diff
@@ -88,12 +88,12 @@ class Process
 end
 
 # stdout of the process is readable
-class IProcess
+class ProcessReader
```

Comment:
I was not sure of that one either, but that was the best I could come up
with.

If someone has a better proposition, I'd be glad to hear it !
On 6 Feb 2015 18:14, "Alexandre Terrasa" <notifications@github.com> wrote:

> In lib/standard/exec.nit
> <https://github.com/privat/nit/pull/1153#discussion_r24279837>:
>
> > @@ -88,12 +88,12 @@ class Process
> >  end
> >
> >  # stdout of the process is readable
> > -class IProcess
> > +class ProcessReader
>
> Not sure for this one but I have nothing else to suggest so, +1
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/privat/nit/pull/1153/files#r24279837>.
>